feature/navigation/drawer/src/main/kotlin/app/k9mail/feature/navigation/drawer/ui/folder/FolderList.kt

```diff
@@ -18,8 +21,9 @@ fun FolderList(
 ) {
     LazyColumn(
         modifier = modifier
-            .fillMaxSize(),
+            .fillMaxWidth(),
     ) {
+        item { Spacer(modifier = Modifier.height(MainTheme.spacings.default)) }
```

Comment:
Instead of having to add `Spacer` items to the list, we could set `contentPadding = PaddingValues(vertical = MainTheme.spacings.default)` on the `LazyColumn`.